tools/build-tools/package.json

```diff
@@ -22,8 +22,10 @@
     "debug:layer-check": "node --inspect-brk dist/layerCheck/layerCheck.js --root ../..",
     "layer-check": "node dist/layerCheck/layerCheck.js --root ../..",
     "list-repo-files": "cd ../.. && git ls-files -co --exclude-standard",
-    "policy-check": "cd ../.. && git ls-files -co --exclude-standard --full-name | node tools/build-tools/dist/repoPolicyCheck/repoPolicyCheck.js -s",
+    "policy-check": "cd ../.. && node tools/build-tools/dist/repoPolicyCheck/repoPolicyCheck.js",
+    "policy-check:debug": "cd ../.. && node --inspect-brk tools/build-tools/dist/repoPolicyCheck/repoPolicyCheck.js ",
     "policy-check:fix": "npm run policy-check -- -r",
+    "policy-check:fix:debug": "cd ../.. && node --inspect-brk tools/build-tools/dist/repoPolicyCheck/repoPolicyCheck.js -r",
```

Comment:
I like the direction, but worry that it will be pain for all engineers. Can / should this step be run as part of release only? And not fail each PR build that added assert?

Reply:
that's how it works currently, but policy-check:fix will tag asserts when run but is optional. plan is to have the person doing the release run it.